sql/atxs/atxs.go

```diff
@@ -288,3 +307,47 @@ func GetAtxIDWithMaxHeight(db sql.Executor) (types.ATXID, error) {
 	}
 	return rst, nil
 }
+
+type CheckpointAtx struct {
+	ID             types.ATXID
+	Epoch          types.EpochID
+	CommitmentATX  types.ATXID
+	VRFNonce       types.VRFPostIndex
+	NumUnits       uint32
+	BaseTickHeight uint64
+	TickCount      uint64
+	SmesherID      types.NodeID
+	Sequence       uint64
+	Coinbase       types.Address
+}
+
+// LatestTwo returns the latest two ATXs per smesher.
+func LatestTwo(db sql.Executor) ([]CheckpointAtx, error) {
```

Comment:
it has unit test. i now expanded it to take an argument N and tested latest 1, 2, and 3.

i decided to stick to this query, instead of doing what you suggested (taking latest, and then its prev) because there are 
two possible approaches:
- parse the whole blob for all miners, then query its previous atx. 
the simplest version is put all the atx blobs into memory. (latest+prevs)
to optimize i do one query with blob (the latest), then one query without blob (prevs).

- add a field in table prev_atx to avoid parsing the whole atx blob into memory
this feels unnecessary as prev_atx isn't part of the checkpointed data